pss-insight-data-sync.py

```diff
@@ -13,10 +13,16 @@
 source_url = os.getenv("SOURCE_URL")
 target_url = os.getenv("TARGET_URL")
 data_elements_url = os.getenv("SOURCE_DATA_ELEMENTS_URL")
+sync_check_url = os.getenv("SYNC_CHECK_URL")
```

Comment:
Given that the path to this configuration is known, instead of fetching from the environment variables, could you please update this to concatenate the base URL with the path to the data store parameters "sync/sync_to_international". 

The base URL is derived from the source_url (https://pssnational.intellisoftkenya.com/api/events)